gunicorn/http/body.py

```diff
@@ -174,10 +174,11 @@ def read(self, size):
         return ret
 
 
-class Body(object):
+class Body(io.IOBase):
     def __init__(self, reader):
         self.reader = reader
         self.buf = io.BytesIO()
+        super().__init__()
```

Comment:
Does this make sense? `IOBase` has no constructor.

Reply:
eh... I don't known how to avoid pylint warning.

Reply:
We can ignore that pylint warning since it doesn't make any sense in most cases.